src/boards/include/boards/pico.h

```diff
@@ -14,21 +14,35 @@
 #ifndef _BOARDS_PICO_H
 #define _BOARDS_PICO_H
 
+// --- UART ---
 #ifndef PICO_DEFAULT_UART
 #define PICO_DEFAULT_UART 0
 #endif
-
 #ifndef PICO_DEFAULT_UART_TX_PIN
 #define PICO_DEFAULT_UART_TX_PIN 0
 #endif
-
 #ifndef PICO_DEFAULT_UART_RX_PIN
 #define PICO_DEFAULT_UART_RX_PIN 1
 #endif
 
+// --- LED ---
 #ifndef PICO_DEFAULT_LED_PIN
 #define PICO_DEFAULT_LED_PIN 25
 #endif
+// no PICO_DEFAULT_WS2812_PIN
+
+// --- I2C ---
+#ifndef PICO_DEFAULT_I2C
+#define PICO_DEFAULT_I2C 0
```

Comment:
Do we _want_ to designate "default I2C" pins for Raspberry Pi Pico, given that there's no such designation of default pins on the pinout diagram at https://www.raspberrypi.org/documentation/pico/getting-started/ ? ping also @aallan

Reply:
this is just to indicate an I2C to use by default

Reply:
In that case I wonder if it might be worth highlighting them with a slightly different colour on https://www.raspberrypi.org/documentation/pico/getting-started/static/15243f1ffd3b8ee646a1708bf4c0e866/Pico-R3-Pinout.svg as we do for the UART function on physical pins 1 and 2? :shrug: